documentation/SA1641.md

```diff
@@ -21,11 +21,12 @@ The file header at the top of a C# code file does not contain the appropriate co
 
 ## Rule description
 
-A violation of this rule occurs when the file header at the top of a C# file does not contain the company name text that has been specified for the project. To enable this rule, navigate to the StyleCop settings for the project and change to the Company Information tab, as shown below:
+A violation of this rule occurs when the file header at the top of a C# file does not contain the company name text that has been specified for the project.
 
-![](Images/CompanyInformationSettings.JPG)
+In order to enable this rule, the `companyName` property must have been specified. In addition to that, the `xmlHeader` property must not have been specified or have a value of **true**.
```

Comment:
I made a minor tweak to the wording here during the merge. In the current wording, some users might interpret this as:

> ...the `xmlHeader` property must not ... have a value of **true**.

After I modified the line, the complete sentence reads as follows:

> In addition to that, the `xmlHeader` property must be either omitted or have a value of **true**.